pkg/features/kube_features.go

```diff
@@ -865,7 +865,7 @@ const (
 )
 
 func init() {
-	runtime.Must(utilfeature.DefaultMutableFeatureGate.Add(defaultKubernetesFeatureGates))
+	runtime.Must(utilfeature.DefaultMutableFeatureGate.Add(defaultKubernetesFeatureGates)) //nolint:forbidigo
```

Comment:
Should this have a TODO with a link to some issue?

```suggestion
	runtime.Must(utilfeature.DefaultMutableFeatureGate.Add(defaultKubernetesFeatureGates)) //nolint:forbidigo // TODO (https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/4330-compatibility-versions): remove this call by making all feature gates versioned
```